bitnami/redis-cluster/7.4/debian-12/prebuildfs/opt/bitnami/scripts/liblog.sh

```diff
@@ -39,7 +38,14 @@ stderr_print() {
 #   None
 #########################
 log() {
-    stderr_print "${CYAN}${MODULE:-} ${MAGENTA}$(date "+%T.%2N ")${RESET}${*}"
+    local prefix=""
+    local color_bool="${BITNAMI_COLOR:-true}"
+    if [[ "$color_bool" = 1 || "$color_bool" =~ ^(yes|true)$ ]]; then
```

Comment:
Let's include the additional options for the checks that are already present in other parts of the file

```
shopt -s nocasematch
```